config_template.cfg

```diff
@@ -1168,6 +1223,21 @@ WRT_CON_FREQ= 1
 % Writing convergence history frequency (dual time, only written to screen)
 WRT_CON_FREQ_DUALTIME= 10
 %
+% Output less information for lower memory use.
+LOW_MEMORY_OUTPUT= NO
+%
+% Write output files (disable all output by setting to NO)
+WRT_OUTPUT= YES
```

Comment:
I still need to put this back in for the FVM solver (it is not being used). Can you please add a comment that this is for the FEM solver only at the moment?

Reply:
@economon, this only applies to WRT_OUTPUT, not to LOW_MEMORY_OUTPUT, right?

Reply:
Yep, just WRT_OUTPUT. I'll come back to this with the other output items soon (like preview binary).